patternfly/patternfly-elements
# Cpfed 4017 a11y tab order primary detail

## Fix Tab Order and Add arrow key features


Fix tab order and add arrow key features.
Add focus indicator and increase the clickable area of links.

### Testing instructions

- Please see **Acceptance Criteria in [this ticket ](https://projects.engineering.redhat.com/browse/CPFED-4080)**
- Using a keyboard tab through to the tab panels and navigate them using the arrow keys.

### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.
